### ISSUE TITLE
Transpiler issue

### ISSUE THREAD

**Issue by kevinforrestconnors** [oven-sh/bun#659]
### Version

0.1.3

### Platform

Linux LAPTOP-ALJK8PUT 5.10.102.1-microsoft-standard-WSL2 #1 SMP Wed Mar 2 00:30:59 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

### What steps will reproduce the bug?

`bun install`
`bun dev`

### How often does it reproduce? Is there a required condition?

Upon launching the server, you will see this error in the console:

```
node_modules.a27efb12fb9ca252.bun:7 Uncaught ReferenceError: Cannot access 'default' before initialization
    at lr (node_modules.a27efb12fb9ca252.bun:7:7023)
    at sr (node_modules.a27efb12fb9ca252.bun:7:7060)
    at extend.js:7:25
```

There is no required condition, the problem has to do with importing [image-js](https://github.com/image-js/image-js).

### What is the expected behavior?

The app should launch with no errors - the error is caused by importing the library.

To verify this, all one has to do is not import the library, and the app runs fine, as there is no other code being run.

### What do you see instead?

In the console:

```
node_modules.a27efb12fb9ca252.bun:7 Uncaught ReferenceError: Cannot access 'default' before initialization
    at lr (node_modules.a27efb12fb9ca252.bun:7:7023)
    at sr (node_modules.a27efb12fb9ca252.bun:7:7060)
    at extend.js:7:25
```

**Comment by FinnRG** [comment]
Could you please provide a minimal reproducible example?

**Comment by oculus42** [comment]
Not able to reproduce this bug with 0.1.4.

**Comment by Electroid** [comment]
This issue has been fixed as of `0.2.2`.

```js
const { Image } = require('image-js');

execute().catch(console.error);

async function execute() {
  let image = await Image.load('cat.jpg');
  let grey = image
    .grey() // convert the image to greyscale.
    .resize({ width: 200 }) // resize the image, forcing a width of 200 pixels. The height is computed automatically to preserve the aspect ratio.
    .rotate(30); // rotate the image clockwise by 30 degrees.
  return grey.save('cat.png', { format: 'png' });
}
```
